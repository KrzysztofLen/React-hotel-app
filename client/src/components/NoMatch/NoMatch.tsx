import React from 'react';
import {Link} from "react-router-dom";

const NoMatch = () => (
    <div className={"noMatch"}>
        <h1 className={"noMatch__header"}>404</h1>
        <h2 className={"noMatch__subHeader"}>Oops! page not found</h2>
        <Link to={"/"} className={"button is-primary"}>Home</Link>
    </div>
);

export default NoMatch;

