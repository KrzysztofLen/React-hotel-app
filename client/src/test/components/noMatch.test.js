import React from 'react';
import {shallow} from 'enzyme';
import NoMatch from '../../components/NoMatch/NoMatch';

test('should render NoMatch component correctly', () => {
	const wrapper = shallow(<NoMatch />);
	expect(wrapper.debug()).toMatchSnapshot();
});