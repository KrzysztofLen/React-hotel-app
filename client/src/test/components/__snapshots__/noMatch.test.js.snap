// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NoMatch component correctly 1`] = `
"<div className=\\"noMatch\\">
  <h1 className=\\"noMatch__header\\">
    404
  </h1>
  <h2 className=\\"noMatch__subHeader\\">
    Oops! page not found
  </h2>
  <Link to=\\"/\\" className=\\"button is-primary\\">
    Home
  </Link>
</div>"
`;
